src/main/rx/cc2500_frsky_shared.c

```diff
@@ -148,14 +151,14 @@ void frSkySpiBind(void)
 static void initialise() {
```

Comment:
Looks like these should be runtime-customizable between FCC and EU versions, now it possibly breaks FCC.

Reply:
it definitely breaks FCC and D8 now. It's wip and even the EU doesn't work reliably yet.